core/src/scss/components/card/_card--icon.scss

```
@@ -0,0 +1,65 @@
+@charset "UTF-8";
+
+.su-card--icon {
+  &,
+  &-font {
+    @include modular-spacing('padding', 2 null null);
+    text-align: center;
+
+    .su-card__contents {
+      @include modular-spacing('padding', 0 null null);
+    }
+
+    &.su-card--minimal {
+      @include padding(0 null null);
+    }
+
+    &.su-card--horizontal {
+      text-align: left;
+
+      @include grid-media-only("xs") {
+        @include grid-column-gap($gutter-xs);
```

Use of deprecated variable.
```suggestion
        @include grid-column-gap($su-gutter-xs);
```